priv/repo/migrations/20200227152752_create_events.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Erlef.Data.Repo.Migrations.CreateEvents do
+  use Ecto.Migration
+
+  def change do
+    create table(:events) do
+    add(:title, :string)
+    add(:slug, :string)
+    add(:datetime, :utc_datetime)
+    add(:start, :utc_datetime)
+    add(:end, :utc_datetime)
+    add(:offset, :string)
+    add(:event_url, :string)
+    add(:gcal_url, :string)
+    add(:gmap_embed_url, :string)
+    add(:organizer, :string)
+    add(:organizer_url, :string)
+    add(:organizer_brand_color, :string)
```

Comment:
@ghislaineguerin Did you want to set a default color here? Let me know and I will add it.

Reply:
```
* title
* slug - but should be generated & unique indexed
* event_type - not mentioned but needed. i.e. Conference, Training, Meetup, etc
* start_date - The start date seems the most important here. If no end date is supplied it could be assumed it's a single day event
* event_url - only required URL
* organizer
* city
* country and/or state
* short_description - limited to 300(?) really it's the length that would work best on the form
```

Some of the other fields I would question the need for them. Example: excerpt, excerpt_html, body, body_html do we really need all these? If you wanted to through

Reply:
I think we can do without those now. They only made sense in the case of markdown data.

Reply:
Ahh yeah, excerpt is the short description, used on an index page, but we can call it description... I don't think we need to put a false constraint on it though. We can limit the length in validation and the form. 

 Now "body" would be the full description, if we want that. Arguably, you could just have description and chop it off after a certain number of chars for the short description.

Reply:
Also yeah, I see some room for normalization, event type being one. I think we should normalize everything properly, but I'm open to suggesting for operating otherwise.

Reply:
@starbelly yes, I wanted to set a default color.

Reply:
@ghislaineguerin what color?